docs/en_US/SQuAD_evolution_examples.md

```diff
@@ -34,22 +20,41 @@ Or Download manually
 
 1. download "dev-v1.1.json" and "train-v1.1.json" in https://rajpurkar.github.io/SQuAD-explorer/
 
-```bash
+```
 wget https://rajpurkar.github.io/SQuAD-explorer/dataset/train-v1.1.json
 wget https://rajpurkar.github.io/SQuAD-explorer/dataset/dev-v1.1.json
 ```
 
 2. download "glove.840B.300d.txt" in https://nlp.stanford.edu/projects/glove/
 
-```bash
+```
 wget http://nlp.stanford.edu/data/glove.840B.300d.zip
 unzip glove.840B.300d.zip
 ```
+### 2.Search Space
+
+Since attention and recurrent neural network (RNN) have been proven effective in Reading Comprehension.
+We conclude the search space as follow:
 
-### 2.2 Update configuration
+1. IDENTITY (Effectively means keep training).
+2. INSERT-RNN-LAYER (Inserts a LSTM. Comparing the performance of GRU and LSTM in our experiment, we decided to use LSTM here.)
+3. REMOVE-RNN-LAYER
+4. INSERT-ATTENTION-LAYER(Inserts an attention layer.)
+5. REMOVE-ATTENTION-LAYER
+6. ADD-SKIP (Identity between random layers).
+7. REMOVE-SKIP (Removes random skip).
+
+![](../examples/trials/ga_squad/ga_squad.png)
+
+### 3.New version
+
```

Comment:
can you confirm what's the corresponding Github Issue for this upcoming "new version"? and add it here.